src/api-service/__app__/onefuzzlib/proxy.py

```diff
@@ -243,8 +271,10 @@ def search_states(cls, *, states: Optional[List[VmState]] = None) -> List["Proxy
         return cls.search(query=query)
 
     @classmethod
+    # Question for Brian - Why does this not include is_used to check forwards?
```

Comment:
Please do not include developer names in code.

Reply:
Apologies. Deleting now.